tests/components/caldav/test_calendar.py

```diff
@@ -1,21 +1,16 @@
 """The tests for the webdav calendar component."""
-# pylint: disable=protected-access
 import datetime
-import logging
-import unittest
-from unittest.mock import (patch, Mock, MagicMock)
+from unittest.mock import MagicMock, Mock
 
-import homeassistant.components.calendar as calendar_base
-import homeassistant.components.caldav.calendar as caldav
+from asynctest import patch
 from caldav.objects import Event
-from homeassistant.const import CONF_PLATFORM, STATE_OFF, STATE_ON
-from homeassistant.util import dt
-from tests.common import get_test_home_assistant
-
-TEST_PLATFORM = {calendar_base.DOMAIN: {CONF_PLATFORM: 'test'}}
+import pytest
 
-_LOGGER = logging.getLogger(__name__)
+from homeassistant.const import STATE_OFF, STATE_ON
+from homeassistant.setup import async_setup_component
+from homeassistant.util import dt
 
+# pylint: disable=redefined-outer-name
```

Comment:
This can probably go away if you specify fixture names in the decorators and call the methods something else.  I.e.  name the method `mock_dav_client_fixture` and specify `name=mock_dav_client` so that when you use the param `mock_dav_client` it's not a redefinition.